### ISSUE TITLE
remark-grid-tables: Incorrectly eaten value

### ISSUE THREAD

**Issue by vhf** [zestedesavoir/zmarkdown#290]
https://sentry.sandhose.fr/zeste-de-savoir/zmarkdown/issues/293/

```
15:1: Incorrectly eaten value
  File "/opt/zmd/node_modules/remark-parse/lib/tokenizer.js", line 194, in validateEat
    new Error(
  File "/opt/zmd/node_modules/remark-parse/lib/tokenizer.js", line 281, in eat
    validateEat(subvalue);
  File "/opt/zmd/node_modules/remark-grid-tables/src/index.js", line 435, in Of.gridTableTokenizer
    return eat(merged)(wrapperBlock)
  File "/opt/zmd/node_modules/remark-parse/lib/tokenizer.js", line 103, in Of.tokenize [as tokenizeBlock]
    method.apply(self, [eat, value]);
  File "/opt/zmd/node_modules/remark-parse/lib/parse.js", line 33, in Of.parse
    children: self.tokenizeBlock(value, content),
...
(22 additional frame(s) were not displayed)

15:1: Incorrectly eaten value: please report this warning on http://git.io/vg5Ft
```

---

Minimal repro:

```md
a   
+--------------------------------+------------+
| b                              | c          |
+--------------------------------+------------+
```

**Comment by vhf** [comment]
@AmarOk1412 Would you please have time to look into this? This exception keeps occurring on Sentry.

**Comment by vhf** [comment]
I seems to come from eol trimming: https://github.com/zestedesavoir/zmarkdown/blob/f705605b99065e80d020eeba2d5f4c74dd0c901a/packages/remark-grid-tables/src/index.js#L232

```js

test('regression: should not crash with leading space', () => {
  const {contents: base} = render(dedent`
    a
    +---+
    | b |
    +---+

    hello
    `)

  const {contents} = render(dedent`
    a·
    +---+
    | b |
    +---+

    hello
    `.replace(/·/g, ' '))

  expect(contents).toBe(base)
})
```

**Comment by artragis** [comment]
@vhf wasn't this issue fixed by #291 ?

**Comment by vhf** [comment]
Yes, thanks!
